<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings_Dialog</class>
 <widget class="QDialog" name="Settings_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>237</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="cbHardwareAcceleration">
     <property name="text">
      <string>Enable Hardware Acceleration</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Middle Click:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ddMiddleClick"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Mouse Wheel:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ddWheel"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cbScrollByWidth">
     <property name="text">
      <string>After Scrolling to Bottom, Scroll by Width</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbRTL">
     <property name="text">
      <string>Right to Left Mode</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbJumpToEnd">
     <property name="text">
      <string>Jump to end of page after scrolling up</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Prevent changing pages for (ms):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tbWaitTime">
       <property name="inputMethodHints">
        <set>Qt::ImhFormattedNumbersOnly</set>
       </property>
       <property name="maxLength">
        <number>4</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Settings_Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Settings_Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
